mindmeld/app.py

```diff
@@ -171,6 +172,20 @@ def _decorator(func):
 
         return _decorator
 
+    def auto_fill(self, **kwargs):
+        """Creates a flow to fill missing entities"""
+
+        def _decorator(func):
+            name = kwargs.pop("name", func.__name__)
```

Comment:
If you go with explicitly-named parameters, here's a common pattern for assigning defaults from a `None` value:
```python
entity_form = name or func.__name__
```

Reply:
I think you mean `name = name or func.__name__`

Reply:
`name = name or func.__name__` will result in a referenced before assignment error here I think.

Also, this behavior was just to keep it consistent with the @app.dialogue_flow decorator. Can change it if you think that's better?

Reply:
Once `name` is defined above in the function signature it will be ok.